src/controller/discoverymgr/mnedc/connectionutil/networkconnectionutil.go

```diff
@@ -24,15 +24,15 @@ import (
 	"github.com/lf-edge/edge-home-orchestration-go/src/restinterface/tls"
 
 	rafftls "github.com/raff/tls-ext"
-	"github.com/raff/tls-psk"
+	psk "github.com/raff/tls-psk"
```

Comment:
Why use the alias `psk`?

Reply:
This is automatically added by the Visual Studion Code. Since there is an implicit use of the `psk` package

Reply:
Understood!